csm_web/frontend/src/components/course/Course.tsx

```diff
@@ -96,9 +99,19 @@ const Course = ({
   /**
    * Render the currently chosen modal.
    */
+
   const renderModal = (): React.ReactElement => {
```

Comment:
The added newline here was probably accidental; it's more conventional to keep the docstring directly attached to the method.

Reply:
duly noted!